_includes/sidebar.html

```diff
@@ -3,10 +3,12 @@
   <li class="sidebar-title">Get started</li>
 
     <ul class="sidebar-items">
-      <li><a href="/setup/">Install</a></li>
-      <li><a href="/getting-started/">Create and run your first app</a></li>
-      <li><a href="/ide-setup/">Setting up the IDE</a></li>
-      <li><a href="/bootstrap-into-dart/">Bootstrap into Dart</a></li>
+      <li><a href="/get-started/">1: Overview</a></li>
+      <li><a href="/get-started/install/">2: Install Flutter</a></li>
+      <li><a href="/get-started/editor-configuration/">3: Editor configuration</a></li>
+      <li><a href="/get-started/test-drive/">4: Test drive</a></li>
+      <li><a href="/get-started/codelab/">5: Core concepts codelab</a></li>
```

Comment:
"Write your first Flutter app". I believe it's much more appealing than "Core concepts codelab", and it's a verb.

Reply:
Done